rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,155 @@
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least) on the available resources:
+
+- Efficient Management Processes, that bias the whole community towards 
+  _coordinated and optimal Resource Employment_.
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+Analysis inspired by [New St.Gallen Managament Model].
+
+[New St.Gallen Managament Model]: https://cio-wiki.org/wiki/The_New_St._Gallen_Management_Model
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which acts as central hub
+for coordinating the community dynamics.
+
+It shall comprehend, consolidate and showcase aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall omit any technical subject about nix or code in general, other than
+repository tooling.
+
+To implement, a detailed review of https://github.com/kubernetes/community
+shall be conducted as joint effort between the interested parties and as 
+an Annex I to this RFC.
+
+Based on this review, a clean repository shall be enacted implementing
+such aspects considered worth porting to our community.
+
+As a general guideline for this review, the vast success and impressive
+dynamics of the kubernetes ecosystem shall be acknowledged not only as a
+proof of concept, but as a very sucessfull and inspiring [viable system].
+
+[viable system]: https://en.wikipedia.org/wiki/Viable_system_theory
+
+## Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas that have sparked interest of RFC commenters
+in their order of submission. They shall be considered _optional_ material
+during implementation, whereas implementors shall have the liberty to
+deliver a consistent experience as they see fit.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+
+# User Stories
+[user-stories]: #user-stories
+
+If this RFC is implemented ...
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can _still_ \* maintain those high quality direct human
+interactions, while having full support of a
+_"[curriculum](https://en.wikipedia.org/wiki/Curriculum)-like"_ resource.
```

Comment:
Perhaps there could be better wording for this? TBH I'm still not sure I get the idea.

Reply:
Is there a wording you would suggest?

The idea is a teacher having a curriculum as oposed to a teacher not having a curriculum. (I don't think teachera without curriculum exist in schooles of western countries).

Reply:
Please unresolve this @blaggacao . My problem is that I don't understand a what's a "curriculum like resource", although I understand what's a curriculum and (I think) I know what's a "resource".

Reply:
> As a mentor, I can inform my efforts with a clear and consistent reference, while maintaining those high quality human interactions, that NixOS community is known for.